sips/sip-019/sip-019-modify-coinbase-reward-per-block-and-halving-schedule.md

```diff
@@ -0,0 +1,74 @@
+# Preamble
+
+SIP Number: 019
+
+Title: Modify Coinbase Reward per Block & Halving Schedule
+
+Author: Xan Ditkoff (email: xan@daemontechnologies.co)
+
+Consideration: Technical, Governance, Economics
+
+Type: Standard
```

Comment:
Per SIP-000, this should be `Type: Consensus`, since it affects the way the whole network operates.  In addition, you'll need to add `Layer: Consensus (hard fork)` beneath it.